frontend/src/libs/packages/services/notification-service.ts

```diff
@@ -0,0 +1,9 @@
+import { toast } from 'react-toastify';
+
+const NotificationService = {
+  error(message: string): void {
```

Comment:
Let's create private show method, that will call toast with ` toast(message, parameters);` and other methods like error will pass message and type in parameters, we will need to create  a enum for notification type